misc/docker-ci/check.mk

```diff
@@ -43,10 +46,13 @@ _do-fuzz-extra:
 	./h2o-fuzzer-http2 -close_fd_mask=3 -runs=1 -max_len=16384 $(SRC_DIR)/fuzz/http2-corpus < /dev/null
 	./h2o-fuzzer-url -close_fd_mask=3 -runs=1 -max_len=16384 $(SRC_DIR)/fuzz/url-corpus < /dev/null
 
+_do-format-check:
+	bash $(SRC_DIR)/misc/format-checker.sh -f clang-format-9 -r origin/master
```

Comment:
I think `bash ` can be omitted?

Or, as an alternative, I think we can simply do everything in `format-check`, rather than splitting the logic into two rules. E.g.,
```
format-check:
    docker run -v `pwd`:/home/source h2oserver/format-checker format-checker.sh -f clang-format-9 -r origin/master
```